### DISCUSSION TITLE
Creating invoice templates fails

### DISCUSSION THREAD

**Discussion by jfbethlehem** [kimai/kimai#3065] [General]
I am trying to create a custom invoice template. The documentation is very well explained and I have followed it, but it does not seem to work (for me).

I added the var/invoices search path to the local.yaml (copied partially from kimai.yaml, uncommented paths and when having a mistake in there, reloading gives an error, so the file is loaded when correct)
I copied a example template from GitHub to var/invoices/mytemplate-pdf.pdf.twig
I edited it and added my logo image (by copying the example code & modifying the path to the logo)
Then I did a bin/console kimai:reload, all green when done
I reloaded the Time Tracking page in the browser when done and did not see the new template there. Logging off and on again, still no new template.

Putting defaults: ~ in the local.yaml also does not remove the default 'Template' invoice template. So I am a bit confused as to why the templates are not loaded....

Only thing I can imagine as to why is that my Kimai install is done through cpanel...it is the latest version though, 1.16.5

**Comment by kevinpapst** [comment]
> I added the var/invoices search path to the local.yaml (copied partially from kimai.yaml, uncommented paths and when having a mistake in there, reloading gives an error, so the file is loaded when correct)

Remove that, not needed!

> I reloaded the Time Tracking page in the browser when done and did not see the new template there. Logging off and on again, still no new template.

Where did you look? It is only visible in the dropdown of one invoice template configuration:
<img width="460" alt="Bildschirmfoto 2022-01-10 um 21 55 17" src="https://user-images.githubusercontent.com/533162/148838073-f0bee631-faf5-421c-85b9-77b06aa5def4.png">
 
> Putting defaults: ~ in the local.yaml also does not remove the default 'Template' invoice template. So I am a bit confused as to why the templates are not loaded....

Remove that as well.

**Reply by jfbethlehem** [reply]
Thanks for the help. I found it. Was rather tricky finding it as it was not where I thought it was.
